### PR TITLE
LPS-115417 increase dynamic data mapping service schema version

### PR DESCRIPTION
**Pull request by marcosapmf**
Related issue: https://issues.liferay.com/browse/LPS-115417

### CODE REVIEW

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
Just started reviewing :)<br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_7_3/UpgradeDDMFormInstanceReport.java [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+					ddmFormInstanceRecords) {
+
+				DDMFormInstanceRecordVersion ddmFormInstanceRecordVersion =
+					_ddmInstanceRecordVersionLocalService.
+						getLatestFormInstanceRecordVersion(
+							ddmFormInstanceRecord.getFormInstanceRecordId(),
+							WorkflowConstants.STATUS_APPROVED);
+
+				dataJSONObject = _processDDMFormValues(
+					_getDDMFormValues(ddmFormInstanceRecordVersion),
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_7_3/UpgradeDDMFormInstanceReport.java:113]: @marcosapmf porque aqui precisou criar uma funcao em vez de usar `ddmFormInstanceRecordVersion.getDDMFormValues()`?

Comment (marcosapmf) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_7_3/UpgradeDDMFormInstanceReport.java:113]: @carolmariaabb, na hora que a gente roda os upgrades, nenhuma classe `*LocalServiceUtil` está inicializada. Daí, quando eu chamo `ddmFormInstanceRecordVersion.getDDMFormValues()`, por exemplo, dá `NullPointerException`, porque dentro desse método tem uma chamada para o `DDMFormInstanceLocalServiceUtil`

**Comment by carolmariaabb** [comment]
@cvanut e/ou @palomaanunes vocês reconhecem esses erros que estão retornando do `ci:test:relevant`?

**Comment by palomaanunes** [comment]
@carolmariaabb , let me check the error and I will let you know as soon as possible

**Comment by palomaanunes** [comment]
> @carolmariaabb , let me check the error and I will let you know as soon as possible

I've just check that the liferay log is thrown NPE, see this [link](https://testray.liferay.com/reports/production/logs/test-1-11/1594225191122/test-portal-acceptance-pullrequest%28master%29/7222/functional-upgrade-tomcat90-mysql57-jdk8_stable/9/0/liferay-log.txt.gz)

**Comment by palomaanunes** [comment]
Liferay

> > @carolmariaabb , let me check the error and I will let you know as soon as possible
> 
> I've just check that the liferay log is thrown NPE, see this [link](https://testray.liferay.com/reports/production/logs/test-1-11/1594225191122/test-portal-acceptance-pullrequest%28master%29/7222/functional-upgrade-tomcat90-mysql57-jdk8_stable/9/0/liferay-log.txt.gz)

this seems to be an error, can you check please?

**Comment by palomaanunes** [comment]
> @cvanut e/ou @palomaanunes vocês reconhecem esses erros que estão retornando do `ci:test:relevant`?

> >@carolmariaabb , let me check the error and I will let you know as soon as possible

> > > I've just check that the liferay log is thrown NPE, see this [link](https://testray.liferay.com/reports/production/logs/test-1-11/1594225191122/test-portal-acceptance-pullrequest%28master%29/7222/functional-upgrade-tomcat90-mysql57-jdk8_stable/9/0/liferay-log.txt.gz)

this seems to be an error, can you check please?

**Comment by carolmariaabb** [comment]
> > @cvanut e/ou @palomaanunes vocês reconhecem esses erros que estão retornando do `ci:test:relevant`?
> 
> > > @carolmariaabb , let me check the error and I will let you know as soon as possible
> 
> > > > I've just check that the liferay log is thrown NPE, see this [link](https://testray.liferay.com/reports/production/logs/test-1-11/1594225191122/test-portal-acceptance-pullrequest%28master%29/7222/functional-upgrade-tomcat90-mysql57-jdk8_stable/9/0/liferay-log.txt.gz)
> 
> this seems to be an error, can you check please?

Thank you @palomaanunes ! I will check it.

**Pull request closed without merging**